fdm-app/app/root.tsx

```diff
@@ -67,8 +73,10 @@ export function Layout() {
                 <Links />
             </head>
             <body>
-                <Outlet />
-                <Toaster />
+                <ErrorBoundary>
+                    <Outlet />
+                    <Toaster />
+                </ErrorBoundary>
```

Comment:
@coderabbitai, I somehow now get always an error page for all pages, even if there is no error and `error` at `ErrorBoundary` is `null`. Do you know how to fix this?

Reply:
I I do that the pages work correctly, but if I simulate an error,  I get a blank page and not the error page. How to solve this?

Reply:
Thanks, but I found a solution already. Please check 2e24d443258293ff3bcbd690f2192ad36e9cb6ec